### PR TITLE
static-neighbor-reports: Update README.md

### PR DESCRIPTION
**Pull request by andybjackson**
Define VAP on first use and fix some typos.

Maintainer: @blocktrron 
Compile tested: n/a readme
Run tested: n/a readme

Description: Define VAP on first use and fix some typos.

Signed-off-by: Andy Jackson <33336318+andybjackson@users.noreply.github.com>

### CODE REVIEW

File: net/static-neighbor-reports/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ the first 6 bytes of the binary neighbor report element.
 The SSID of the foreign AP. This option can be omitted, in case it matches the SSID used on the local AP.
```
Comment (dcousens) [inline net/static-neighbor-reports/README.md:27]: A few lines up, `s/followign/following`

File: net/static-neighbor-reports/README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ the first 6 bytes of the binary neighbor report element.
...
 
 #### iface
-Space seperated list of hostapd interfaces the neighbor should be installed to.
+Space separated list of hostapd interfaces the neighbor should be installed to.
 
 ## Retrieving neighbor information
 To retrieve the neighbor informations of an AP to be isntalled on a foreign AP, make sure the UCI option
```
Comment (dcousens) [inline net/static-neighbor-reports/README.md:33]:
```suggestion
To retrieve the neighbor information of an AP to be installed on a foreign AP, make sure the UCI option
```

File: net/static-neighbor-reports/README.md (hunk 1 above)
Comment (andybjackson) [reply net/static-neighbor-reports/README.md:27]: Good catch, thanks!

**Review by andybjackson — COMMENTED** [review]
Enhanced to pass checks

**Pull request closed without merging**